src/InboxNotifications.php

```diff
@@ -74,9 +74,10 @@ public static function create_surface_cart_checkout_blocks_notification() {
 			return;
 		}
 
-		$data_store = \WC_Data_Store::load( 'admin-note' );
-		$note_ids   = $data_store->get_notes_with_name( self::SURFACE_CART_CHECKOUT_NOTE_NAME );
-
+		if ( ! apply_filters( 'woocommerce_admin_disabled', false ) ) {
```

Comment:
I wonder if we should simply return early if `woocommerce_admin_disabled` is true. Something along these lines:

```PHP
if ( apply_filters( 'woocommerce_admin_disabled', false ) ) {
	return;
}
```

If WC Admin is disabled, I don't think we need to continue running the logic that creates the Notification, because it will never show up for users, right?